lib/tasks/several_schedules.rake

```diff
@@ -6,8 +6,8 @@ namespace :data do
   task move_events_attributes: :environment do
     Program.all.each do |program|
       schedule = Schedule.create(program: program)
-      program.selected_schedule = schedule
-      program.save
+      program.selected_schedule = schedule unless program.selected_schedule
```

Comment:
did `selected_schedule` exist in openSUSE? Before there was only one schedule, so this attribute was not there. Was it?

Reply:
or is this in case you run it after another schedule has been created? That makes sense...

Reply:
the latter yes